packages/expo-notifications/ios/EXNotifications/Permissions/EXUserFacingNotificationsPermissionsRequester.m

```diff
@@ -19,10 +19,10 @@ + (NSString *)permissionType
   return @"userFacingNotifications";
 }
 
-- (instancetype)initWithMethodQueue:(dispatch_queue_t)methodQueue
+- (instancetype)init
```

Comment:
the header file should account for this update as well

Reply:
Done.